packages/backend/src/open_payments/quote/model.ts

```diff
@@ -20,7 +20,8 @@ export class Quote extends WalletAddressSubresource {
       'receiveAmount',
       'minExchangeRate',
       'lowEstimatedExchangeRate',
-      'highEstimatedExchangeRate'
+      'highEstimatedExchangeRate',
+      'method'
```

Comment:
we should also be able to expect the `method` field as an argument to quote creation

Reply:
I'm not sure if I fully understand so please clarify if I have something wrong. Given that this is an objection model, these quotes objects are created from inserting and querying the database. Rather than doing something like `new Quote({..., method: 'ilp'})`. So I cant pass it in to the quote creation because it doesnt exist in the database and [I think we agreed that the database changes didn't need to be in this PR](https://github.com/interledger/rafiki/pull/2054#discussion_r1364740532). And if its defined as a real class attribute, it will try to write `ilp` to a non-existant `method` column where ever we insert a quote.

At face value I think it would be appropriate to include all of that here (add to `quoteService.create`, adjust everywhere that's called, add migration). However I think we're sacrificing debugging of `nl-open-payments-updates` at this point to do so, and it's functional as-is. I can capture this improvement in a task if we like, and work on that if we have extra time, otherwise do it after the code-freeze.

Reply:
I think what Max meant was adding `method` to the arguments of `quoteService.create`.

Reply:
We can, but it will be unused until we make the db changes